index.rst

```diff
@@ -35,111 +27,363 @@ Times of events
-   +-------------+--------------------------+--------------------------+
-   | All on 50%  | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
-   | All on 60%  | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
-   | All on 70%  | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
-   | All on 80%  | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
-   | All on 90%  | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
-   | All on 100% | '2023-12-07 14:33:00.00' | '2023-12-07 14:50:00.00' |
-   +-------------+--------------------------+--------------------------+
+   +---------+------------------------+------------------------+
+   |FCU state| Start Time             | End Time               |
+   +=========+========================+========================+
+   |       0 | 2023-12-07 14:20:00.00 | 2023-12-07 14:30:00.00 |
+   +---------+------------------------+------------------------+
+   |      10 | 2023-12-07 14:55:30.00 | 2023-12-07 14:59:55.00 |
+   +---------+------------------------+------------------------+
+   |      20 | 2023-12-07 15:00:05.00 | 2023-12-07 15:04:00.00 |
+   +---------+------------------------+------------------------+
+   |      30 | 2023-12-07 15:05:05.00 | 2023-12-07 15:10:00.00 |
+   +---------+------------------------+------------------------+
+   |      40 | 2023-12-07 15:12:30.00 | 2023-12-07 15:14:30.00 |
+   +---------+------------------------+------------------------+
+   |      50 | 2023-12-07 15:16:15.00 | 2023-12-07 15:20:00.00 |
+   +---------+------------------------+------------------------+
+   |      60 | 2023-12-07 15:21:40.00 | 2023-12-07 15:25:00.00 |
+   +---------+------------------------+------------------------+
+   |      70 | 2023-12-07 15:26:05.00 | 2023-12-07 15:30:00.00 |
+   +---------+------------------------+------------------------+
+   |      80 | 2023-12-07 15:31:05.00 | 2023-12-07 15:40:00.00 |
+   +---------+------------------------+------------------------+
+   |      90 | 2023-12-07 17:48:55.00 | 2023-12-07 17:50:10.00 |
+   +---------+------------------------+------------------------+
+   |     100 | 2023-12-07 14:33:00.00 | 2023-12-07 14:45:00.00 |
+   +---------+------------------------+------------------------+
+   |     100 | 2023-12-07 17:51:30.00 | 2023-12-07 17:53:00.00 |
+   +---------+------------------------+------------------------+
+
+Data quality
+============
+We have checked that the TMA was not moving during this test (Azimuth: -25.95 deg - Elevation: 89.95 deg) and that the mirror was raised (MTM1M3.logevent_detailedState == ACTIVEENGINEERING) 
```

Comment:
I recommend adding single-back-quotes here too: 
`MTM1M3.logevent_detailedState == ACTIVEENGINEERING`

Reply:
Same as above --> we made a block of code